iml-wire-types/src/lib.rs

```diff
@@ -1935,19 +1976,34 @@ impl From<&str> for LdevEntry {
                 .get(3)
                 .unwrap_or_else(|| panic!("LdevEntry must specify a device.")))
             .to_string(),
+            fs_type: (*parts
```

Comment:
Lot's of clippy warnings here.

Reply:
I had to run `cargo clean`. I see them now.